server/models/Profile.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require("mongoose");
+const User = require("User");
+const profileSchema = new mongoose.Schema({
+  firstname: {
+    type: String,
+    required: true,
+  },
+  lastname: {
```

Comment:
Prefer camelCase for your variable names

Reply:
Thank you, I have made the necessary changes